i18n/da.json

```diff
@@ -0,0 +1,580 @@
+    "subscribers.email": "E-mail",
+    "subscribers.emailExists": "E-mail findes allerede.",
+    "subscribers.errorBlocklisting": "Fejl ved blokering af abonnenter: {error}",
+    "subscribers.errorNoIDs": "Ingen ID'er givet.",
+    "subscribers.errorNoListsGiven": "Ingen lister givet.",
+    "subscribers.errorPreparingQuery": "Fejl under forberedelse af abonnentforespørgsel: {error}",
+    "subscribers.errorSendingOptin": "Fejl ved afsendelse af tilmeldings-e-mail.",
+    "subscribers.export": "Eksport",
+    "subscribers.invalidAction": "Ugyldig handling.",
+    "subscribers.invalidEmail": "Ugyldig e-mail.",
+    "subscribers.invalidJSON": "Ugyldig JSON i attributter.",
+    "subscribers.invalidName": "Ugyldigt navn.",
+    "subscribers.listChangeApplied": "Listeændring anvendt.",
+    "subscribers.lists": "Lister",
+    "subscribers.listsHelp": "Lister, som abonnenterne selv har afmeldt sig fra, kan ikke fjernes.",
+    "subscribers.listsPlaceholder": "Lister at abonnere på",
+    "subscribers.manageLists": "Administrer lister",
+    "subscribers.markUnsubscribed": "Markér som afmeldt",
+    "subscribers.newSubscriber": "Ny abonnent",
+    "subscribers.numSelected": "{antal} valgte abonnent(er)",
+    "subscribers.optinSubject": "Bekræft abonnement",
+    "subscribers.preconfirm": "Bekræft abonnementer på forhånd",
+    "subscribers.preconfirmHelp": "Send ikke opt-in-e-mails, og markér alle listeabonnementer som 'abonnerede'.",
+    "subscribers.query": "Forespørgsel",
+    "subscribers.queryPlaceholder": "E-mail eller navn",
+    "subscribers.reset": "Nulstil",
+    "subscribers.selectAll": "Vælg alle {num}",
+    "subscribers.sendOptinConfirm": "Send tilmeldingsbekræftelse",
+    "subscribers.sentOptinConfirm": "Tilmeldingsbekræftelse sendt",
+    "subscribers.status.blocklisted": "Blokeret",
+    "subscribers.status.confirmed": "Konfirmeret",
+    "subscribers.status.enabled": "Aktiveret",
+    "subscribers.status.subscribed": "Abonnerede",
+    "subscribers.status.unconfirmed": "Ubekræftet",
+    "subscribers.status.unsubscribed": "Afmeldt",
+    "subscribers.subscribersDeleted": "{num} abonnent(er) udgår",
+    "templates.cantDeleteDefault": "Kan ikke slette ikke-eksisterende eller standardskabelon",
+    "templates.default": "Standard",
+    "templates.dummyName": "Dummy-kampagne",
+    "templates.dummySubject": "Dummy-kampagneemne",
+    "templates.errorCompiling": "Fejl ved kompilering af skabelon: {error}",
+    "templates.errorRendering": "Fejlmeddelelse om fejlgengivelse: {error}",
+    "templates.fieldInvalidName": "Ugyldig længde for navn.",
+    "templates.makeDefault": "Indstil standard",
+    "templates.newTemplate": "Ny skabelon",
+    "templates.placeholderHelp": "Pladsholderen {placeholder} skal vises nøjagtigt én gang i skabelonen.",
+    "templates.preview": "Forhåndsvisning",
+    "templates.rawHTML": "Rå HTML",
+    "templates.subject": "Emne",
+    "users.login": "Log ind",
+    "users.logout": "Log ud",
```

Comment:
This trailing comma is breaking the JSON and should be removed. Last issue to fix!

You can validate JSON using a site like https://www.jsonlint.com/